demography.md

```diff
@@ -29,34 +29,56 @@ from IPython.display import SVG
 
 ## Population structure
 
+All population structure and demography is passed to {func}`msprime.sim_ancestry` via a {class}`msprime.Demography` object.
+
+```{code-cell} ipython3
```

Comment:
Just FYI, we don't need the ``ipython3`` tags here

Reply:
I wonder if this preamble is much help here - would it be better to just skip to the first example and then explain that?